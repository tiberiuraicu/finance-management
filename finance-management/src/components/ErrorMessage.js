import React from "react";
import { Text } from "react-native";

const ErrorMessage = (props) => {
  function errorMessageColor(errorTruthValue) {
    return {
      color: !errorTruthValue ? "red" : "transparent",
      marginLeft: "5%",
      height: "5%",
    };
  }
  return (
    <Text
      style={{ ...errorMessageColor(props.errorTruthValue), ...props.style }}
    >
      {props.children}
    </Text>
  );
};

export default ErrorMessage;
