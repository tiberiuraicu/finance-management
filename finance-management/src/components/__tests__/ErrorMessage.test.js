import ErrorMessage from "../ErrorMessage";
import { shallow } from 'enzyme';

describe('ErrorMessage component tests', () => {
    it('renders the component', () => {
      const wrapper = shallow(<ErrorMessage />);
      expect(wrapper).toMatchSnapshot();
    });
  });