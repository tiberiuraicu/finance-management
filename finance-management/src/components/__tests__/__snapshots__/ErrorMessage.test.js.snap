// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage component tests renders the component 1`] = `ShallowWrapper {}`;
